### ISSUE TITLE
Dataset used for network training

### ISSUE THREAD

**Issue by XuxinChen8** [SSinyu/RED_CNN#9]
Hi, I have learned a lot going through your code! What confuses me a little bit is the dataset used but it is not your problem. Previous publications including Red_CNN all unanimously used 2378 full and quarter dose 512*512 CT images from **10 patients**. However, the dataset I downloaded has **50 patients** in total. So I am wondering if the original dataset provided at the "_2016 NIH-AAPM-Mayo Clinic Low Dose CT Grand Challenge_" has been expanded. Could you help me to see which 10 patients are used for training your network? Thank you very much!

Below is 50 patients IDs of my downloaded dataset:
![Screenshot from 2020-07-05 09-15-09](https://user-images.githubusercontent.com/61339991/86534821-2ea81180-bea1-11ea-8c0c-000ff84b0c72.png)

**Comment by youngjun-ko** [comment]
The training and testing patient indices would be:
L067, L096, L109, L143, L192, L286, L291, L310, L333 (training) and L506 (testing).

**Comment by strive160** [comment]
May I ask where the data set you mentioned was downloaded? There is no such data as L067 mentioned above in the list I downloaded

**Comment by youngjun-ko** [comment]
Aforementioned 10 patient numbers are the lists of original dataset at the "_2016 NIH-AAPM-Mayo Clinic Low Dose CT Grand Challenge_".

**Comment by strive160** [comment]
In the challenge I opened, the whole data set has 952G, and there is no data of these patients you mentioned. Could you please provide the data download link of these patients?
I don't know if I made a mistake.
Thank you very much!

**Comment by strive160** [comment]
Is the data of the above 10 patients still available

**Comment by strive160** [comment]
> 上述10位患者编号是“ _2016 NIH-AAPM-Mayo诊所低剂量CT大型挑战_”的原始数据集。

Can you share the data set link

**Comment by XuxinChen8** [comment]
It would be really helpful if someone can provide the dataset link for the aforementioned 10 patients!

**Comment by ztt0821** [comment]
> It would be really helpful if someone can provide the dataset link for the aforementioned 10 patients! I had to change to work on another project just because I am not able to get the original _2016 NIH-AAPM-Mayo_ dataset.......

Me too!!! I write email to MayoClinic. They give me some IDs, but just from test set, not the training set. So sad....

**Comment by Yudha-David** [comment]
The dataset is available in the link: https://aapm.app.box.com/s/eaw4jddb53keg1bptavvvd1sf4x3pe9h/folder/144226105715.
